aspnetcore/signalr/publish-to-azure-web-app.md

```diff
@@ -1,63 +1,70 @@
+   | **Name**           | Unique name of the app. |
+   | **Subscription**   | Azure subscription that the app uses. |
+   | **Resource Group** | Group of related resources to which the app belongs. |
+   | **Hosting Plan**   | Pricing plan for the web app. |
 
-| Item | Description |
-| ---- | ----------- |
-| **App name** | A unique name of the app. |
-| **Subscription** | The Azure subscription that the app uses. |
-| **Resource Group** | The group of related resources to which the app belongs.  |
-| **Hosting Plan** | The pricing plan for the web app. |
+1. Select the **Azure SignalR Service** in the **Dependencies** > **Add** drop-down list:
 
-![Create app service](publish-to-azure-web-app/_static/create-app-service-dialog.png)
+   ![Dependencies area showing the selection of Azure SignalR Service in the Add drop-down list](publish-to-azure-web-app/_static/signalr-service-dependency.png)
+
+1. In the the **Azure SignalR Service** dialog, select **Create a new Azure SignalR Service instance**. Provide a **Name**, **Resource Group**, and **Location**. Return to the **Azure SignalR Service** dialog and select **Add**.
 
 Visual Studio completes the following tasks:
 
 * Creates a Publish Profile containing publish settings.
 * Creates or uses an existing *Azure Web App* with the provided details.
 * Publishes the app.
-* Launches a browser, with the published web app loaded.
-
-Notice the format of the URL for the app is *{app name}.azurewebsites.net*. For example, an app named `SignalRChattR` has a URL that looks like `https://signalrchattr.azurewebsites.net`.
+* Launches a browser, which loads the web app.
 
-If an HTTP 502.2 error occurs, see [Deploy ASP.NET Core preview release to Azure App Service](xref:host-and-deploy/azure-apps/index) to resolve it.
+The format of the app's URL is `{App Service Name}.azurewebsites.net`. For example, an app named `SignalRChatApp` has a URL of `https://signalrchatapp.azurewebsites.net`.
 
-## Configure SignalR web app
+If an HTTP *502.2 - Bad Gateway* error occurs during a preview release, see [Deploy ASP.NET Core preview release to Azure App Service](xref:host-and-deploy/azure-apps/index#deploy-aspnet-core-preview-release-to-azure-app-service) to resolve it.
 
-ASP.NET Core SignalR apps that are published as an Azure Web App must have [ARR Affinity](https://en.wikipedia.org/wiki/Application_Request_Routing) enabled. [WebSockets](xref:fundamentals/websockets) should be enabled, to allow the WebSockets transport to function.
+## Configure the app in Azure App Service
 
-In the Azure portal, navigate to **App Settings** for your web app. Set **WebSockets** to **On**, and verify **ARR Affinity** is **On**.
+[ARR Affinity](https://azure.github.io/AppService/2016/05/16/Disable-Session-affinity-cookie-(ARR-cookie)-for-Azure-web-apps.html) is required (the default is **On**). Enable [WebSockets](xref:fundamentals/websockets) to allow the WebSockets transport to function (the default is **Off**).
```

Comment:
I believe these steps are only needed when you are *not* using the Azure SignalR Service

Reply:
@bradygaster @anurse Can you guys confirm?

Reply:
This is correct. If you **are using** the Azure SignalR Service you **do not** need WebSockets nor ARR Affinity enabled on your app. This is because the clients actually connect their WebSockets to the Service, not directly to your app.